steward/src/cork.rs

```diff
@@ -139,3 +284,11 @@ async fn send_cork(cork: Cork) -> Result<TxResponse, Error> {
     .await
     .map_err(|e| e.into())
 }
+
+async fn contract_call<T: Detokenize>(contract_call: ContractCall<EthSignerMiddleware, T>) {
+    let gas = contract_call.estimate_gas().await.unwrap();
+    let gas_price = CellarGas::get_gas_price().await.unwrap();
+    let contract_call = contract_call.gas(gas).gas_price(gas_price);
+    let contract_call = contract_call.send().await.unwrap();
```

Comment:
We need to account for error conditions from executing this contract call, so we can't just `unwrap` it. We'll have to wait for it to resolve and return a `Result`.